publish-mqtt/src/homie.rs

```diff
@@ -171,10 +183,30 @@ impl HomieDevice {
             HOMIE_VERSION,
         )
         .await?;
+        // TODO: Send $localip and $mac too.
```

Comment:
is this a TODO for part of this PR, or a later PR? Does it also want to be tracked on the board, or is it fine to stay as just a TODO in the code?

Reply:
For a later PR. Up to you how you want to manage tasks, I can add it if you want.